### PR TITLE
Update to use new POST endpoint in calculation orchestrator

### PR DESCRIPTION
**Pull request by paulsimonandrews**
https://eaflood.atlassian.net/browse/FCEP-71

Change the call to calculate payments to use the new endpoint

### CODE REVIEW

File: server/services/payments-service.js [hunk 1]
```diff
@@ -3,24 +3,17 @@ const wreck = require('@hapi/wreck').defaults({
...
-      action: { id },
-      options: { quantity }
-    }]
-  }
-}
-
 async function calculatePayment (parcelRef, actionId, quantity) {
+  const payloadData = JSON.stringify({ quantity })
+  console.log('In updated calculatePayment')
+  console.log(payloadData)
```
Comment (jaucourt) [inline server/services/payments-service.js]: Did you mean to leave in these as logs?

**Review by jaucourt — COMMENTED** [review]
Just one little thing, otherwise LGTM.

File: server/services/payments-service.js (hunk 1 above)
Comment (paulsimonandrews) [reply server/services/payments-service.js]: No, they are removed in a local commit, so will be pushed before review :-)

**Review by jaucourt — APPROVED** [review]
LGTM

**Pull request merged**